src/language-css/parser-postcss.js

```diff
@@ -390,6 +385,60 @@ function parseNestedCSS(node, options) {
       node.value = parseValue(value);
     }
 
+    if (node.type === "css-atrule") {
+      // mixin
+      if (node.mixin) {
+        const source =
+          node.raws.identifier +
+          node.name +
+          node.raws.afterName +
+          node.raws.params;
+        node.selector = parseSelector(source);
+        delete node.params;
+        return node;
+      }
+
+      // function
+      if (node.function) {
+        return node;
+      }
+
+      // only css support custom-selector
+      if (node.name === "custom-selector" && options.parser === "css") {
+        const customSelector = node.params.match(/:--\S+?\s+/)[0].trim();
+        node.customSelector = customSelector;
+        node.selector = parseSelector(
+          node.params.slice(customSelector.length).trim()
+        );
+        delete node.params;
+        return node;
+      }
+
+      // Whitespace between variable and colon
+      if (node.name.indexOf(":") !== -1 && !node.params) {
```

Comment:
We should improve this check, Scss allow to use `:` in interpolations and can't have params, so this invalid